go/vt/vttablet/tabletserver/repltracker/repltracker.go

```diff
@@ -130,6 +131,7 @@ func (rt *ReplTracker) Status() (time.Duration, error) {
 
 	switch {
 	case rt.isPrimary || rt.mode == tabletenv.Disable:
+		replicationLagSeconds.Reset() // we are the primary, we have no lag
```

Comment:
I could go either way about this being necessary compared to the first call. I don't think it's incorrect (both clauses imply we don't want these metrics), but I could still go either way.